service-seed-without-router/play-service/conf/routes

```diff
@@ -10,5 +10,5 @@ GET  /:service/health                      @controllers.health.HealthController.
 # Logs Management APIs
 # POST    /v1.3/system/log/update           @controllers.logsmanager.LogController.setLogLevel()
 
-GET  /echo/:id                                 @controllers.EchoController.echo(id:String)
+GET  /echo/:id                                 @controllers.EchoController.echo(id:String,request: play.mvc.Http.Request)
```

Comment:
Point 1: 
GET  /:service/health                      @controllers.health.HealthController.getUserOrgServiceHealth(service:String)
  move all health releated code inside health Controller only. THis ia a seeder project it should not have any notion about  user and org. 

Point 2: I don't think echo api is required , as we already have health

Reply:
https://github.com/indrajra/play-service-seeder/pull/9#discussion_r344063639